docs/modules/ROOT/pages/how-tos/cloudscale/enable-loadbalancer-service.adoc

```diff
@@ -0,0 +1,365 @@
+= Enable `LoadBalancer` serivces on cloudscale.ch
+
+[abstract]
+--
+Steps to enable `LoadBalancer` serivces on an OpenShift 4 cluster on https://cloudscale.ch[cloudscale.ch].
+--
+
+[NOTE]
+--
+- In the future, this procedure will be mostly automated
```

Comment:
Remove that, doesn't add any value and when it gets automated, the documentation will get updated as well.